test/blackbox-tests/test-cases/display.t

```diff
@@ -2,7 +2,9 @@ Document how Dune displays various things
 =========================================
 
   $ echo '(lang dune 3.0)' > dune-project
-  $ export BUILD_PATH_PREFIX_MAP=SH=`command -v sh`
+  $ 
```

Comment:
empty

Reply:
Not sure what you mean, but here is the latest code:

```
  $ ENCODED_SH=$(dune_cmd encode-prefix "`command -v sh`")
  $ export BUILD_PATH_PREFIX_MAP="SH=$ENCODED_SH"
```

I've gone through all the places that set BUILD_PATH_PREFIX_MAP manually, and made sure that all of the paths are properly encoded using the new "dune_cmd encode-prefix" command.

When I run `command -v sh` on either Linux or MSYS2's bash, I get `/usr/bin/sh`. But perhaps on some Windows environments this could give a Windows-like path that would need encoding.